src/backend/dx12/src/command.rs

```diff
@@ -742,8 +742,8 @@ impl CommandBuffer {
                         let cut_width = cmp::min(r.image_extent.width, cut_row_texels as image::Size);
                         let gap_texels = (row_offset - aligned_offset) as image::Size / image.bytes_per_block as image::Size;
                         // this is a conservative row pitch that should be compatible with both copies
-                        let max_unaligned_pitch = r.image_extent.width * image.bytes_per_block as u32;
-                        let row_pitch = (max_unaligned_pitch | d3d12::D3D12_TEXTURE_DATA_PITCH_ALIGNMENT) + 1;
+                        let max_unaligned_pitch = (r.image_extent.width + gap_texels) * image.bytes_per_block as u32;
```

Comment:
What's the motivation behind this one?

Reply:
The pitch is not really used here, since we copy row by row, but we need to make it at least as big as the rows we copy (to make DX12 happy). `r.image_extent.width + gap_texels` is the upper bound for those 2 copies to follow